src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/MethodologyServiceTests.cs

```diff
@@ -515,6 +516,253 @@ await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId)
+                Title = "Publication title"
+            };
+
+            var topic = new Topic
+            {
+                Title = "Topic title",
+                Publications = ListOf(publication)
+            };
+
+            var theme = new Theme
+            {
+                Title = "Theme title",
+                Topics = ListOf(topic)
+            };
+
+            var latestVersions = ListOf(
+                new MethodologyVersion
+                {
+                    Id = Guid.NewGuid(),
+                    MethodologyContent = new MethodologyVersionContent {
+                        Annexes = new List<ContentSection>(),
+                        Content = new List<ContentSection>(),
+                    },
+                    PreviousVersionId = null,
+                    PublishingStrategy = Immediately,
+                    Status = Approved,
+                    AlternativeTitle = "Methodology 1 v0 title",
+                    Version = 0,
+                    Methodology = new Methodology
+                    {
+                        Slug = "methodology-1-slug",
+                    }
+                },
+                new MethodologyVersion
+                {
+                    Id = Guid.NewGuid(),
+                    MethodologyContent = new MethodologyVersionContent {
+                        Annexes = new List<ContentSection>(),
+                        Content = new List<ContentSection>(),
+                    },
+                    PreviousVersionId = null,
+                    PublishingStrategy = Immediately,
+                    Status = Approved,
+                    AlternativeTitle = "Methodology 2 v0 title",
+                    Version = 0,
+                    Methodology = new Methodology
+                    {
+                        Slug = "methodology-2-slug"
+                    }
+                });
+
+            var contentDbContextId = Guid.NewGuid().ToString();
+
+            await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId))
+            {
+                await contentDbContext.Themes.AddAsync(theme);
+                await contentDbContext.SaveChangesAsync();
+            }
+            
+            var methodologyVersionRepository = new Mock<IMethodologyVersionRepository>(MockBehavior.Strict);
+
+            methodologyVersionRepository.Setup(mock => mock.GetLatestPublishedVersionByPublication(publication.Id))
+                .ReturnsAsync(latestVersions);
+
+            await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId))
+            {
+                var service = SetupMethodologyService(contentDbContext,
+                    methodologyVersionRepository: methodologyVersionRepository.Object);
+
+                var result = await service.GetSummariesTree();
+                VerifyAllMocks(methodologyVersionRepository);
+
+                // Assert that the cached result is what was returned by the method.
```

Comment:
I think this comment has been copied from `MethodologyServiceCachingTests` but isn't applicable anymore.